recipes/fiji/20170530/meta.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: fiji
+  version: 20170530
+
+source:
+  -   url: https://downloads.imagej.net/fiji/Life-Line/fiji-linux64-20170530.zip  # [linux]
+      md5: 1aaa2d11daac043cb0e63d00c8c79253   # [linux]
+  -   url: https://downloads.imagej.net/fiji/Life-Line/fiji-macosx-20170530.dmg # [osx]
+      md5: a5c9af6c0ced968931faa3280e2394e1   # [osx]
+
+build:
+  number: 0
+
+requirements:
+  host:
+    - openjdk
+  run:
+    - openjdk
```

Comment:
@gregvonkuster I think you can add python as runtime dep here, if jython is a use-case.